src/compiler/Restler.Compiler.Test/swagger/schemaTests/swagger_escape_characters.json

```diff
@@ -14,15 +14,15 @@
             "required": true,
             "in": "body",
             "schema": {
-              "$ref": "#/definitions~1Escape~0Characters"
+              "$ref": "#/definitions/Escape~0Characters"
```

Comment:
>ions/Escape~0Cha [](http://example.com/codeflow?start=32&length=16)

@justdanpo Thank you for the fix!  Apologies for the delay in merging this.  Just one quick question, why did you modify these existing refs - are they not valid?  If they are still expected to work, could you please keep them and add separate operations for the new ones?

Reply:
`"$ref": "#/definitions~1Escape~0Characters"` points to
```json
{
  "definitions/Escape~Characters": {
    "properties": {
      "escape_characters1": {
        "type": "string"
      }
   }
}
```
which does not exist in the schema.
Correct reference `"$ref": "#/definitions/Escape~0Characters"` points to
```json
{
  "definitions": {
    "Escape~Characters": {
      "properties": {
        "escape_characters1": {
          "type": "string"
        }
      }
    }
}
```